### ISSUE TITLE
Using latest docker image, buildarr seems to not be using config Host: value for connecting.

### ISSUE THREAD

**Issue by gleepwurp** [buildarr/buildarr#176]
Hi!
Even though I have the following buildarr.yml configuration with an FQDN as a host, buildarr tries to use the config's key (sonarr) in order to connect to my instance, instead of the actual host.

Config:
```
...
sonarr:
  host: "sonarr.fizban.ca"
  port: 443
  protocol: "https"
```

buildarr execution log:

```
buildarr  | urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='sonarr', port=443): Max retries exceeded with url: /api/v3/system/status (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f61ca6a3910>: Failed to resolve 'sonarr' ([Errno -3] Try again)"))

```
It's picking of the either or both the https protocol and port 443, but not the host: value correcly.

buildarr  | 2024-02-14 19:52:57,200 buildarr:1 buildarr.cli.daemon [INFO] Buildarr version 0.7.1 (log level: INFO)

**Comment by Callum027** [comment]
Hi there, apologies for not getting to this sooner (I was taking a break for a while).

I'm glad you figured out the problem, and I apologise for the inaccurate documentation causing you problems. I'll fix this ASAP.

**Comment by Callum027** [comment]
The website has now been corrected, and all instance configuration examples now use the correct field (`hostname`).

Thank you for reporting this problem. All feedback I get helps make Buildarr easier to use for others, so if you run into any more issues, please create a new ticket on this repository.

**Comment by gleepwurp** [comment]
Thank you! Looking forward to using it... Waiting on the Sonarr v4 integration to finish putting it in!
